lib/binance.ex

```diff
@@ -400,6 +402,40 @@ defmodule Binance do
     create_order(symbol, "SELL", "MARKET", quantity)
   end
 
+  defp parse_order_response({
```

Comment:
If you just want to parse a map into a struct, you can use [exconstructor](https://github.com/appcues/exconstructor) which is already a dep of this project

Add `use ExConstructor` to your struct (like [here](https://github.com/dvcrn/binance.ex/blob/master/lib/binance/symbol_price.ex#L11)

Then use `.New()` to parse (like [here](https://github.com/dvcrn/binance.ex/blob/master/lib/binance.ex#L179))